outline.md

```diff
@@ -0,0 +1,90 @@
+# An Oriole for Inner Source
+45 min
+
+## Abstract
+
+_Note: This abstract was created in February 2018 for submission of this talk to [the ISC Spring Summit](http://paypal.github.io/InnerSourceCommons/events/isc-spring-2018-agenda/) and is already posted there_.
+
+Inner source is the application of open source methodologies to internally-developed software.
+While simple to define, inner source can be difficult to explain and implement successfully.
+This presentation introduces the key terms, concepts, and principles for effective inner sourcing along with explanations and real examples.
+It is content that is accessible to newcomers while at the same time refreshing to those with experience.
+The same information in this presentation is also delivered as an O’Reilly Oriole, available freely at https://www.safaribooksonline.com/oriole/.
+We’ll review the Oriole, how to use it, and other Orioles related to inner source that are on the way.
+
+## Background
+Many individuals that are excited about inner source share their experience in [the InnerSource Commons](http://innersourcecommons.org/) and other public locations.
+In my experience it's been reasonably difficult for inner source practitioners to share ideas and guidance that are directly usable by others.
+
+## Goals
+The goal of this presentation is to share guidelines, structure, and vocabulary for the public conversation around inner source.
+We'll present common terms that all can use when describing inner source.
+Having this common vocabulary is a tangible first step to speaking, working, and building things that all inner source practitioners can use.
+
+We'll also present the opportunity to work on building the further learning paths funded by PayPal.
+After this presentation, the attendees should understand:
+
+* the key terms and principles shared in the learning path.
+* how to view the learning path videos.
+* how to express interest in working on any future segments in the learning path.
+
+## Audience
+This presentation will be given at [the ISC Spring Summit](http://paypal.github.io/InnerSourceCommons/events/isc-spring-2018-agenda/).
+It is intended for anyone interested in inner source whether novice or expert.
+
+## Outline
+
+### Introduction
+
+* It's been a fantastic conference.
+* I'm excited for others not in this room to get the same learning.
+* There's not a one-stop, public location for concrete, approachable, and actionable information on inner source.
+* Inner source is becoming a buzz word.  Need to get ahead of the brand dilution.
```

Comment:
Why is it important to get ahead of brand dilution?    e.g. The worst case scenario, someone says oh we've tried InnerSource and didn't work for us.  Now it is a dead strategy for that company and people can't get the benefits.